recipes-graphics/mesa/mesa-gl_%.bbappend

```diff
@@ -0,0 +1 @@
+PACKAGES_remove = "${@bb.utils.contains("MACHINE_FEATURES", "sgx", "libgbm libgbm-dev", "", d)}"
```

Comment:
maybe we should fix mesa recipe instead of this hack?

Reply:
So producing empty libgbm and libgbm-dev instead of not building them at all is a bug in mesa - not the other's inability to handle empty packages. Then yes, fixing mesa is the most correct approach. Will try this. Thanks!